src/main/java/qna/domain/Question.java

```diff
@@ -75,21 +56,31 @@ public boolean isOwner(User loginUser) {
         return writer.equals(loginUser);
     }
 
-    public Question setDeleted(boolean deleted) {
-        this.deleted = deleted;
-        return this;
-    }
-
     public boolean isDeleted() {
         return deleted;
     }
 
-    public List<Answer> getAnswers() {
-        return answers;
+    public boolean deletable(User loginUser) {
+        if (!isOwner(loginUser)) {
+            return false;
+        }
+
+        return answers.isOwner(loginUser);
+    }
+
+    public List<DeleteHistory> delete() {
```

Comment:
```suggestion
    public List<DeleteHistory> delete(User loginUser) {
```

위와 같이 deletable() 메서드를 통합하는 것은 어떨까?